### DISCUSSION TITLE
Is it possible to finetune an fp8 or int 4 flux model?

### DISCUSSION THREAD

**Discussion by Vigilence** [kohya-ss/sd-scripts#1731] [Q&A]
Instead of the full fp16 model, can we use am fp8 or int4 to finetune?

**Comment by johnr14** [comment]
`If you specify --fp8_base, you can use fp8 models for FLUX.1. The fp8 model is only compatible with float8_e4m3fn format.`

Use sd3 branch, the info is all there.
